tests/instancetype/instancetype.go

```diff
@@ -238,6 +241,90 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 			memRequest := vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory]
 			Expect(memRequest.Value()).To(Equal(expectedOverhead))
 		})
+		It("should apply 80% memory overcommit from instancetype without a preference", func() {
+			vmi := libvmifact.NewAlpine()
+
+			instancetype := builder.NewInstancetypeFromVMI(vmi)
+			instancetype.Spec.Memory.OvercommitPercent = 80
+
+			instancetype, err := virtClient.VirtualMachineInstancetype(testsuite.GetTestNamespace(instancetype)).
+				Create(context.Background(), instancetype, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			vm := libvmi.NewVirtualMachine(vmi,
+				libvmi.WithInstancetype(instancetype.Name),
+				libvmi.WithRunStrategy(virtv1.RunStrategyAlways),
+			)
+			vm, err = virtClient.VirtualMachine(testsuite.GetTestNamespace(vm)).
+				Create(context.Background(), vm, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			Eventually(matcher.ThisVM(vm)).WithTimeout(timeout).WithPolling(time.Second).
+				Should(matcher.BeReady())
+
+			vmi, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(vmi)).
+				Get(context.Background(), vm.Name, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			Expect(*vmi.Spec.Domain.Memory.Guest).To(Equal(instancetype.Spec.Memory.Guest))
+
+			expectedRequest := int64(float32(instancetype.Spec.Memory.Guest.Value()) * (1 - float32(instancetype.Spec.Memory.OvercommitPercent)/100))
```

Comment:
It might make sense to put that calculation into a helper.

Reply:
Ack, there’s another possibility that 
We could move all common code into smaller helper functions, like createInstancetypeFromVMI, createPreference, createVM, and expectedMemoryRequest.
or should i just split the calculation?